test/sensor/Component.cdef

```diff
@@ -13,7 +13,6 @@ sources:
 
 cflags:
 {
-    -I${CURDIR}/../../components/periodicSensor/
```

Comment:
You could remove the whole cflags section to simplify the .cdef file.

Reply:
☑️ Done